Apps/phsentinelone/sentinelone_connector.py

```diff
@@ -648,6 +666,39 @@ def _get_agent_id(self, search_text, action_result):
         else:
             return response['data'][0]['id']
 
+    def _get_site_id(self, action_result):
+        header = self.HEADER
+        header["Authorization"] = "APIToken %s" % self.token
+        site_val, response = self._make_rest_call('/web/api/v2.1/sites', action_result, headers=header, method='get')
+        if phantom.is_fail(site_val):
+            return str(-1)
+        endpoints_found = len(response['data']['sites'])
+        self.save_progress("Endpoints found: {}".format(str(endpoints_found)))
+        action_result.add_data(response)
+        if endpoints_found == 0:
+            return '0'
+        elif endpoints_found > 1:
+            return '99'
+        else:
+            return response['data']['sites'][0]['id']
+
+    def _get_download_id(self, search_text, action_result):
+        header = self.HEADER
+        header["Authorization"] = "APIToken %s" % self.token
+        download_id, response = self._make_rest_call('/web/api/v2.1/activities?limit=100&sortBy=createdAt&sortOrder=desc&skip=0', action_result,
+            headers=header, method='get')
+        if phantom.is_fail(download_id):
+            return str(-1)
+        endpoints_found = len(response['data'])
```

Comment:
change the variables names as required in _get_download_id and _get_site_id. 
And also check for the possiblity of the multiples values in this cases.
Don' we require multiple values in case of sites ?

Reply:
we do not require multiple values for sites.

Reply:
But what if there exists multiple site? What step we are taking for that case?

Reply:
I have checked the API document. when I run '/web/api/v2.1/sites' this in postman and in SentinelOne it gives only one site information

Reply:
Because we have credentials which have access to one site. What if the user have access to multiple sites?